portfolio/src/main/webapp/script.js

```diff
@@ -56,10 +61,151 @@ function deleteComments(){
   generateComments(0);
 }
 
-function createMap() {
-  const map = new google.maps.Map(
+function dayMode() {
+  map = new google.maps.Map(
       document.getElementById('map'),
-      {center: {lat: 37.422, lng: -122.084}, zoom: 16});
+      {center: {lat: 37.7590, lng: -77.4800}, zoom: 7});
+
+    if(markersVisible){
+            markers();
+        }
+}
+
+function nightMode(){
+    // Styles a map in night mode.
+        map = new google.maps.Map(document.getElementById('map'), {
+          center: {lat: 37.7590, lng: -77.4800},
+          zoom: 7,
+          styles: [
```

Comment:
Do you need to re-create the entire map just to change the style and zoom?  There's probably a way to do this with the existing map